### ISSUE TITLE
Problem with toJson

### ISSUE THREAD

**Issue by sergejostir** [samturrell/laroute#3]
Hello.

I'm running L6.18.35 and there seems to be a problem with toJson method.
https://github.com/samturrell/laroute/blob/bede1a47dacebf458a88298df1411707b6c1dcf2/src/Routes/Collection.php#L16-L21

If I comment this part out, it's working. If I do not, I get the following:

```
Symfony\Component\Debug\Exception\FatalThrowableError  : Argument 1 passed to Lord\Laroute\Routes\Collection::__construct() must be an instance of Illuminate\Routing\RouteCollection, array given, called in xxx\vendor\laravel\framework\src\Illuminate\Support\Collection.php on line 640

  at C:\Users\Sergej\Apps\laragon\www\panel\vendor\lord\laroute\src\Routes\Collection.php:12

  Exception trace:

  1   Lord\Laroute\Routes\Collection::__construct()
      xxx\vendor\laravel\framework\src\Illuminate\Support\Collection.php:640

  2   Illuminate\Support\Collection::map(Object(Closure))
      xxx\vendor\laravel\framework\src\Illuminate\Support\Traits\EnumeratesValues.php:719
```

**Comment by samturrell** [comment]
Hi @sergiz - could you check version `2.0.0`? A PR was just submitted to fix this (see #4).

Let me know if the problem persists.

**Comment by sergejostir** [comment]
I don't use this package anymore, but I believe you that it got fixed.
